com.st.STM32CubeIDE.yaml

```diff
@@ -0,0 +1,72 @@
+app-id: com.st.STM32CubeIDE
+runtime: org.gnome.Platform
+runtime-version: '3.36'
+sdk: org.gnome.Sdk
+sdk-extensions: 
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: stm32cubeide
+finish-args:
+  - --socket=wayland
```

Comment:
Does it actually work under Wayland or just uses xwayland?

Reply:
I'm not actually sure and went after what Eclipse supports. I'll change it to just using x11 for the time being.